server/routes/detailsOfOffence/deleteOffence.test.ts

```diff
@@ -66,6 +69,89 @@ beforeEach(() => {
     },
   })
 
+  decisionTreeService.adjudicationData.mockResolvedValue({
+    draftAdjudication: {
+      id: 102,
+      adjudicationNumber: 1524493,
+      prisonerNumber: 'G6415GD',
+      incidentDetails: {
+        locationId: 197682,
+        dateTimeOfIncident: '2021-12-09T10:30:00',
+        handoverDeadline: '2021-12-11T10:30:00',
+      },
+      incidentRole: {
+        roleCode: undefined,
+      },
+      offenceDetails: [
+        {
+          offenceCode: 1,
+          victimPrisonersNumber: 'G5512G',
+        },
+        {
+          offenceCode: 2,
+        },
+      ],
+      startedByUserId: 'TEST_GEN',
+    },
+    adjudicationNumber: 102,
+    incidentRole: undefined,
+    prisoner: {
+      offenderNo: undefined,
+      firstName: 'ADJUDICATION_PRISONER_FIRST_NAME',
+      lastName: 'ADJUDICATION_PRISONER_LAST_NAME',
+      categoryCode: undefined,
+      language: undefined,
+      friendlyName: undefined,
+      displayName: undefined,
+      prisonerNumber: undefined,
+      currentLocation: undefined,
+      assignedLivingUnit: undefined,
+    },
+    associatedPrisoner: undefined,
+  })
+
+  decisionTreeService.allOffences.mockResolvedValue([
+    {
+      victimOtherPersonsName: undefined,
+      victimPrisonersNumber: 'G5512G',
+      victimStaffUsername: undefined,
+      offenceCode: '2001',
+    },
+  ])
+
+  const twoQuestions = [
+    {
+      question:
+        'Assisted: Adjudication_prisoner_first_name Adjudication_prisoner_last_name. Associated: Adjudication_associated_prisoner_first_name Adjudication_associated_prisoner_last_name',
+      answer: 'Prisoner victim: A_prisoner_first_name A_prisoner_last_name',
+    },
+    {
+      question:
+        'Assisted: Adjudication_prisoner_first_name Adjudication_prisoner_last_name. Associated: Adjudication_associated_prisoner_first_name Adjudication_associated_prisoner_last_name',
+      answer: 'A standard answer with child question',
+    },
+    {
+      question: 'A child question',
+      answer: 'A standard child answer',
+    },
+  ]
+
+  const singleQuestion = [
+    {
+      question:
+        'Assisted: Adjudication_prisoner_first_name Adjudication_prisoner_last_name. Associated: Adjudication_associated_prisoner_first_name Adjudication_associated_prisoner_last_name',
+      answer: 'Prisoner victim: A_prisoner_first_name A_prisoner_last_name',
+    },
+  ]
+
+  decisionTreeService.questionsAndAnswers
```

Comment:
This seems to fix the issue I was having with the test, but I feel like it defeats the original purpose of the test? Suggestions welcome!